scripts/balances.py

```diff
@@ -0,0 +1,63 @@
+import dbs
+import bson
+
+golos_accounts = dbs.golos_db['account_object']
+
+def save_balances(doc):
+    cyberway_accounts = dbs.cyberway_db['accounts']
+    cyberway_balances = dbs.cyberway_db['balances']
+
+    if (cyberway_accounts.find({"_SCOPE_":doc["name"]}).count() or
+            cyberway_balances.find({"_SCOPE_":doc["name"]}).count()):
+        return
+
+    balance = {
+	"balance": {
+	    "amount": doc["balance_value"],
+	    "decs": bson.Int64(4),
+	    "sym": "SYS"
+	},
+	"_SCOPE_": doc["name"],
+	"_PAYER_": doc["name"],
+	"_SIZE_": bson.Int64(16)
+    }
+    dbs.cyberway_db['accounts'].save(balance)
```

Comment:
Balance must save into `eosio.token` smart-contract. This contract has own database: `_CYBERWAY_eosio-token`